rmw_connext_cpp/src/rmw_service.cpp

```diff
@@ -56,7 +56,7 @@ rmw_create_service(
     return nullptr;
   }
 
-  auto node_info = static_cast<ConnextNodeInfo *>(node->data);
+  auto node_info = static_cast<ConnextParticipantInfo *>(node->data);
```

Comment:
@ivanpauno here and elsewhere, `ConnextNodeInfo` became `ConnextParticipantInfo`, but I don't see the `struct` defined and `node_info` is probably not the right name anymore.

Reply:
The struct is defined [here](https://github.com/ros2/rmw_connext/blob/4e2e037593cf27fa1e1201bb333fb2e7680a1971/rmw_connext_shared_cpp/include/rmw_connext_shared_cpp/types.hpp#L129).

And about renaming `node_info` to `participant_info`, I did that in https://github.com/ros2/rmw_connext/pull/382/commits/4e2e037593cf27fa1e1201bb333fb2e7680a1971.